src/app/clubs/[id]/events/series/[seriesId]/analytics/page.tsx

```
@@ -0,0 +1,156 @@
+            } finally {
+                setIsLoading(false);
+            }
+        }
+        fetchData();
+    }, [seriesId]);
+
+    const handleRecalculateAll = async () => {
+        setIsRecalculating(true);
+        try {
+            // Trigger recalculation for all attendees
+            await Promise.all(
```

ESLint

[error] Replace `············` with `······`

(prettier/prettier)